samples/snippets/test_sample_pagination.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import typing
+
+from sample_pagination import print_images_list, print_images_list_by_page
+
+PROJECT = 'windows-sql-cloud'
+
+
+def test_pagination(capsys: typing.Any) -> None:
+    print_images_list(PROJECT)
+    out, _ = capsys.readouterr()
```

Comment:
In this test we wouldn't be able to tell if they are delivered in pages of same size, we don't have access to pages using this method.

In the next test with `print_images_list_by_page` we can do that.

I wouldn't validate the images names, as those can change at any time. What we care here for is that the results are delivered and, in case of the second test, are divided into pages.

As for ordering - I don't think API guarantees any order of the responses. I think you can specify ordering while doing a request, but we don't do it here to keep the sample as simple as possible.

Reply:
Sounds good!